pages/code_schools.js

```diff
@@ -1,71 +1,172 @@
-import Head from 'components/head';
+import Select from 'react-select';
 import { getCodeSchoolsPromise } from 'common/constants/api';
+import Head from 'components/head';
+import HeroBanner from 'components/_common_/HeroBanner/HeroBanner';
 import OutboundLink from 'components/_common_/OutboundLink/OutboundLink';
 import Section from 'components/_common_/Section/Section';
+import Button from 'components/_common_/Button/Button';
 import SchoolCard from 'components/Cards/SchoolCard/SchoolCard';
 import { s3 } from 'common/constants/urls';
+import States from 'common/constants/dropdown-states-values';
+import styles from './styles/code_schools.css';
 
 export default class CodeSchools extends React.Component {
   state = {
-    schools: [],
+    allSchools: [],
+    filteredSchools: [],
+    selectedStates: [],
   };
 
   async componentDidMount() {
     const { data } = await getCodeSchoolsPromise();
-    this.setState({ schools: data });
+    this.setState({ allSchools: data });
```

Comment:
you can do this in the same line:
```suggestion
    this.setState({ allSchools: data, filteredSchools: data });
```

Reply:
Yes, all the styles are still screwed up. I just wanted to get your opinion on general layout.